docs/BUILD

```diff
@@ -17,6 +17,7 @@ filegroup(
             "root/**/*.pb",
         ],
         exclude = [
+            "root/**/envoy-dynamic*.yaml",
```

Comment:
same comment as on other PR (https://github.com/envoyproxy/envoy/pull/13783#discussion_r514217718) - its not ideal to exclude these configs, but its not trivial to include either